Source/WebCore/bindings/js/SerializedScriptValue.h

```diff
@@ -62,6 +62,10 @@ class MemoryHandle;
 } }
 #endif
 
+namespace JSC {
+class ErrorInstance;
```

Comment:
I'm a little surprised JSGlobalObject doesn't need a forward declaration too.  May as well add it?

Reply:
Added.